HadesScheduler/go.mod

```
@@ -1,6 +1,8 @@
 module github.com/ls1intum/hades/hadesScheduler
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.3
```

Why that? I would prefer to just fix it to `go 1.22` because with this, we get automatic minor bug fix updates without the necessity to change anything

```suggestion
go 1.22
```